BDArmory/Armor/BDAdjustableArmor.cs

```diff
@@ -349,26 +224,27 @@ void UpdateThickness(bool onLoad = false)
                 return;
             }
             if (onLoad) return; //don't adjust part placement on load
+            /*
             if (armorthickness != Oldthickness)
             {
                 float ratio = (armorthickness - Oldthickness) / 100;
-                foreach (Part p in part.children)
+
```

Comment:
Disabled for now, due to the old version having issues with edge detection, and that remains true now; would need effective method of detecting a part has been surface attached to the armor panel faces and not edges.